src/pages/assessmentModule/AssessmentModule.jsx

```diff
@@ -36,7 +36,8 @@ const AssessmentModule = ({ type }) => {
   const [formErrors, setFormErrors] = useState({});
   const [weightageErrors, setWeightageErros] = useState({});
   const [assigneeSearch,setAssigneeSearch] = useState("");
-  const [isAssigneeLoading,setIsAssigneeLoading] = useState(false)
+  const [isAssigneeLoading,setIsAssigneeLoading] = useState(false);
+  const [isAssessmentLoading,setIsAssessmentLoading] = useState(false);
```

Comment:
instead using this many usestate, declare a object in usestate try to update key based on need.

Reply:
Need disscussion